test/controllers/test_controller_test.rb

```diff
@@ -493,6 +493,49 @@ class TestControllerTest < ActionController::TestCase
     Rails.unstub(:env)
   end
 
+  test "should create standalone fact check and associate with the team" do
+    # Test setup
+    team = create_team
+    user = create_user
+    create_team_user(user: user, team: team)
+
+    get :create_imported_standalone_fact_check, params: {
+      team_id: team.id,
+      email: user.email,
+      description: 'Test description',
+      context: 'Test context',
+      title: 'Test title',
+      summary: 'Test summary',
+      url: 'http://example.com',
+      language: 'en'
+    }
```

Comment:
Suggestion for a stronger test: Wrap this whole block in a `assert_difference 'FactCheck.count' do ... end` block.

Reply:
:white_check_mark: